core/templates/services/png-sanitizer.service.spec.ts

```diff
@@ -0,0 +1,71 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit test for PngSanitizationService.
+ */
+
+import { HttpClientTestingModule } from '@angular/common/http/testing';
+import { TestBed } from '@angular/core/testing';
+import { DomSanitizer } from '@angular/platform-browser';
+import { PngSanitizerService } from './png-sanitizer.service';
+
+
+
+describe('PngSanitizerService', () => {
```

Comment:
Can you provide some context as to why this is needed as part of a refactor? Changing PNGs to be sent via base64 seems like a behavioral change, not just a refactor. Why is that necessary?